src/map.rs

```diff
@@ -271,10 +373,6 @@ impl Map {
                         // don't draw player if invisible
                         continue;
                     }
-                    let tile_offset = vec2(
-                        tile_x * constants::WORLD_SIZE,
-                        tile_y * constants::WORLD_SIZE,
-                    );
```

Comment:
(OwO) what is this change?

Reply:
This code has moved to when radar objects are added (instead of here where they are drawn)